src/Guides/RestructuredText/Parser/DocumentIterator.php

```diff
@@ -49,6 +49,16 @@ public function key(): int
         return $this->position;
     }
 
+    /**
+     * @deprecated Work around for Production's eating one line too many
+     * @todo Revisit The Loop in {@see DocumentParser::parseLines()} and see if the Look Ahead timing should be done
+     *       differently
+     */
+    public function prev(): void
```

Comment:
The doctrine lexer has a lookAhead method with a special pointer which can be increased by each look ahead. Would that fit here as well? 
I think you had this intention with `getNextLine`

Reply:
I want to fix this in a subsequent PR; the prev is evil and should be gone but I think this is big enough as is :D